### PR TITLE
reformat, remove notice and add hook

### PR DESCRIPTION
**Pull request by nevergone**
- Auto-reformat code
- remove unnecessary line with PHP notice
- hook_google_oauth_create_user_alter() added

### CODE REVIEW

File: src/Controller/GoogleOAuthController.php [hunk 1]
```diff
@@ -4,93 +4,93 @@
...
+    }
+
+    $this->client = new Google_Client();
+    $this->client->setAuthConfigFile($config_file);
+    $this->client->setScopes(array('email'));
+    $this->client->setState('offline');
+
+    // Set the redirect URL which is used when redirecting and verifying
+    // the one-time oauth code.
+    $uri = \Drupal::url('google_oauth.authenticate', array(), array('absolute' => TRUE));
```
Comment (brettsaguid) [inline src/Controller/GoogleOAuthController.php:34]: Old array syntax here and line 29

Comment (nevergone) [reply src/Controller/GoogleOAuthController.php:34]: Old array syntax is allowed: https://www.drupal.org/coding-standards#array

Comment (brettsaguid) [reply src/Controller/GoogleOAuthController.php:34]: Mkay 👍  everything else looks good to me

**Comment by brettsaguid** [comment]
Guess Im not allowed to approve it :/ sorry about that

File: google_oauth.api.php [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+
+/**
+ * @file
+ * Describe hooks provided by the Google OAuth module.
+ */
+
+/**
+ * @addtogroup hooks
+ * @{
```
Comment (bendoh) [inline google_oauth.api.php:11]: TIL.

**Comment by bendoh** [comment]
This looks great, thanks for your cleanup @nevergone! This was my very first D8 module and I wrote it during a session "Building your first Drupal 8 module" – I'm happy to see it getting a little traction!

I will add unit tests in the future, too, but this PR really gives it a little polish. Thanks again!

**Pull request merged**